lib/jazzy/source_declaration/type.rb

```diff
@@ -61,6 +61,14 @@ def extension?
         kind =~ /^source\.lang\.swift\.decl\.extension.*/
       end
 
+      def extensible?
```

Comment:
Now that we have both ObjC & Swift types in there, I've prefixed the other language-specific methods with `objc_` and `swift_`. Could you please do the same for `extensible?` and `protocol?`.